flink-dstl/flink-dstl-dfs/src/main/java/org/apache/flink/changelog/fs/DuplicatingStateChangeFsUploader.java

```diff
@@ -51,12 +52,12 @@
  *       <li>Store the meta of files into {@link ChangelogTaskLocalStateStore} by
  *           AsyncCheckpointRunnable#reportCompletedSnapshotStates().
  *       <li>Pass control of the file to {@link LocalChangelogRegistry#register} when
- *           ChangelogKeyedStateBackend#notifyCheckpointComplete() , files of the previous
- *           checkpoint will be deleted by {@link LocalChangelogRegistry#discardUpToCheckpoint} at
- *           the same time.
+ *           ChangelogKeyedStateBackend#buildSnapshotResult , files of the previous checkpoint will
```

Comment:
Control transfer is no longer when ChangelogKeyedStateBackend#buildSnapshotResult

Reply:
Fixed.